classes/Learnosity/Shortcodes/js/init-items.js

```diff
@@ -1,6 +1,7 @@
-signed_requests.forEach(signed_request => {
+signed_requests.forEach((signed_request, index) => {
     window.learnosityCollection = window.learnosityCollection || [];
     window.learnosityCollection.push(LearnosityItems.init(JSON.parse(signed_request), {
+        readyListener: new Function(ready_listeners[index]),
```

Comment:
[sanitize_text_field](https://github.com/Learnosity/wordpress-learnosity/pull/5/commits/e752a74ed9e95d7a45937b9719c63d21ea13ee12#diff-c8c3780c65f3e27fe6bea6e5ae56b5541669e919e44333e1d3d987f02435c24bR47) is called on the ready listener specified for the `lrn-items` or `lrn-assess` shortcode. The ready listener is extracted from the said shortcode as a string so I had to use `Function()` to somewhat "convert" it to JS code so that it gets executed. I didn't use `eval()` because I read online that it should be avoided. 😅